apps/teams-test-app/src/components/privateApis/ExternalAppCardActionsAPIs.tsx

```diff
@@ -43,7 +43,7 @@ const ProcessActionSubmit = (): React.ReactElement =>
 const ProcessActionOpenUrl = (): React.ReactElement =>
   ApiWithTextInput<{
     appId: string;
-    url: string;
+    url: URL;
```

Comment:
So what should we do when we are trying to input url? Can we still use string ('') to input? if not, what should we do?

Reply:
Hmm, yeah this should still be string, right? And then just convert to URL?

Reply:
That's what I think so. Otherwise the input will not be valid, I guess.